README.md

```diff
@@ -644,7 +644,7 @@ A startup goes through [several stages of growth](#stages-of-a-startup) as it ra
 based on the hope and expectation that the company will grow and make more money in the
 future.
 
-🄳 Companies add (or “issue”) shares during fundraising, which can then be awarded to
+🄳 Companies add (or “issue”) shares during fundraising, which can then be traded to
```

Comment:
Revert on this PR.

"Traded" has wrong connotations (e.g. trading floors, or that there is a broker). Brad used "exchange," which is more accurate. But making that requires additional rewording, probably "can be exchanged for cash from investors". But regardless, that change is not a copy edit and needs to be a different PR.

Reply:
K. Reverting for now and will issue a new PR.